docs/esi-leap-requirements.md

```diff
@@ -0,0 +1,15 @@
+# MOC esi-leap Requirements
+
+## Overview
+esi-leap is the leasing part of [Elastic Secure Infrastructure](https://github.com/CCI-MOC/esi) (esi), an OpenStack service for acting as a FLOCX provider to communicate with a FLOCX marketplace. It is intended to work on top of the [esi-common library](https://github.com/CCI-MOC/esi-common), which supports an OpenStack install that simulates Ironic multi-tenancy through the use of `project_owner_id` and `project_id` property attributes.
```

Comment:
esi-leap is its own complete Openstack service, not just a part of esi, as far as I understand it

Reply:
Also from the documentation I have read, esi-leap should be stylized ESI-Leap

Reply:
Got it!

Reply:
Agreed - let's talk about esi-leap as an independent leasing service first, and mention the connection to esi later.

esi-common *should* also no longer be needed; now Ironic nodes have 'owner' and 'lessee' attributes that replace the esi-common solution.